Tools/cases_generator/flags.py

```diff
@@ -40,11 +42,19 @@ def fromInstruction(instr: parsing.Node) -> "InstructionFlags":
             )
             and not has_free,
             HAS_EVAL_BREAK_FLAG=variable_used(instr, "CHECK_EVAL_BREAKER"),
+            HAS_DEOPT_FLAG=variable_used(instr, "DEOPT_IF"),
+            HAS_ERROR_FLAG=(
+                variable_used(instr, "ERROR_IF")
+                or variable_used(instr, "error")
+                or variable_used(instr, "pop_1_error")
+                or variable_used(instr, "exception_unwind")
+                or variable_used(instr, "resume_with_error")
```

Comment:
We could make ``variable_used`` accept a sequence of strings to check.

Reply:
We could, but why bother? Plus, I'd kind of like `variable_used(instr, X, Y, Z)` to look for the *sequence* of identifiers `X Y Z`. (And I'd also like to allow keywords, e.g. `goto`.)